core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -274,8 +296,8 @@ class Main {
                          */
                         //assert(linesInfo.total <= totalLines){ "WRONG COVERAGE: ${linesInfo.total} > $totalLines"}
 
-                        info("Covered targets (lines, branches, faults, etc.): ${targetsInfo.total}")
-                        info("Potential faults: ${faults.size}")
+                        //info("Covered targets (lines, branches, faults, etc.): ${targetsInfo.total}")
+                        //info("Potential faults: ${faults.size}")
```

Comment:
the fix here for this PR was as simple as moving this line before the if check on BB

Reply:
Hi Andrea, thank you so much for your comment. I have just done that. I think I overcomplicated the issue assuming that the issue is for all the information just because I saw non-zero numbers for targets and coverage in statistics.csv file for black-box. However, I checked the file again and both are 0 for black-box. In that case, I just fixed the issue by moving the line 

` info("Potential faults: ${faults.size}") `

before 

`if (!config.blackBox || config.bbExperiments) {`